packages/core/stories/divider/divider.stories.tsx

```diff
@@ -1,10 +1,10 @@
 import { StoryFn } from "@storybook/react";
 import { StackLayout } from "@salt-ds/core";
 
-import { Divider, DividerProps } from "@salt-ds/lab";
+import { Divider, DividerProps } from "@salt-ds/core";
```

Comment:
This should be merged into the import above

Reply:
done